_posts/2015-02-09-create-your-application.markdown

```diff
@@ -64,134 +63,135 @@ For more information about generating a Rails application see the [Getting Start
 
   1. Add GeoBlacklight to your `Gemfile`
 
-    ```ruby
-    # In ./Gemfile
-    gem 'blacklight'
-    gem 'geoblacklight'
-    gem 'jettywrapper'
-    ```
+     ```ruby
+     # In ./Gemfile
+     gem 'blacklight'
+     gem 'geoblacklight'
+     gem 'jettywrapper'
+     ```
 
   1. Install required gems and their dependencies
 
-    ```sh
-    $ bundle install
-    ```
+     ```sh
+     $ bundle install
+     ```
 
   1. Run Blacklight generator (with devise authentication)
 
-    ```sh
-    $ rails g blacklight:install --devise
-    ```
+     ```sh
+     $ rails g blacklight:install --devise
+     ```
 
   1. Run GeoBlacklight generator (with a jetty instance for Solr)
 
-    ```sh
-    $ rails g geoblacklight:install --jettywrapper
-    ```
+     ```sh
+     $ rails g geoblacklight:install --jettywrapper
+     ```
 
   1. Run database migrations
 
-    ```sh
-    $ rake db:migrate
-    ```
+     ```sh
+     $ rake db:migrate
+     ```
+
+
+     Quick tip: All of these tasks (1 - 5) are included as part of template to generate a new GeoBlacklight application. To run that generator just run:
+     {: .flash-notice}
 
-    <div class='flash-notice'>
-      Quick tip: All of these tasks (1 - 5) are included as part of template to generate a new GeoBlacklight application. To run that generator just run:
-      
+     ```sh
+     $ rails new your_app_name -m https://raw.githubusercontent.com/geoblacklight/geoblacklight/master/template.rb
+     ```
```

Comment:
:+1: Sweet.. now this? Any ideas?

<img width="755" alt="screen shot 2016-02-18 at 4 10 01 pm" src="https://cloud.githubusercontent.com/assets/1656824/13158263/2c647618-d65a-11e5-8278-2da4e352b44a.png">